packages/fints/jest.config.json

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
Why was this file added but the configuration for jest not removed in the package.json? Either delete the file or the section in the package.json please :)